tiledbimg/converters/base.py

```diff
@@ -150,6 +155,12 @@ def to_tiledb(
     def _write_image(
         self, uri: str, image: np.ndarray, metadata: Dict[str, Any]
     ) -> None:
+        """
```

Comment:
No need to document (every) private methods (even more so when there's no actual docstring)